pod/perldelta.pod

```diff
@@ -366,7 +366,29 @@ manager will later use a regex to expand these into links.
 
 =item *
 
-XXX
+Certain constructs involving a two-variable C<for> loop would crash the perl
+compiler in v5.42.0:
+
```

Comment:
The SHA the perldelta refers to is going to change once this is merged.

I used to add the SHA in when I did separate PRs/commits after the fix itself was merged, but it doesn't work when the perldelta is included in the PR :(